core/model/resource.py

```diff
@@ -565,7 +565,7 @@ class Representation(Base, MediaTypes):
     image_width = Column(Integer, index=True)
 
     # The content of the representation itself.
-    content = Column(Binary)
+    content = Column(LargeBinary)
```

Comment:
Do these turn into the same underlying postgresql data type as before?

Reply:
Yes this is the same data type as before. https://docs.sqlalchemy.org/en/14/changelog/changelog_14.html#change-214869a306f72f33237772d33fc332ec